scripts/postgres_support.sh

```diff
@@ -5,9 +5,9 @@ echo "Postgres support : ${POSTGRES}"
 if [ "$POSTGRES" == True ]; then
  pip3 install psycopg2-binary
  #Enable postgres support
- sed -i '/# Sqlite3 suport/,/# End Sqlite3 support/d' ../MobSF/settings.py && \
- sed -i "/# Postgres DB - Install psycopg2/,/'''/d" ../MobSF/settings.py && \
- sed -i "/# End Postgres support/,/'''/d" ../MobSF/settings.py && \
+ sed -i '/# Sqlite3 suport/,/# End Sqlite3 support/d' ../MobSF/settings.py && \  
+ sed -i ':a;N;$!ba;s/# Postgres DB - Install psycopg2\n"""/# Postgres DB - Install psycopg2/g' ../MobSF/settings.py && \  
```

Comment:
@superpoussin22 What does `:a;N;$!ba;s` do ?

Reply:
without that it is  impossible to remove the """ in settings.py